server/src/main.rs

```diff
@@ -570,6 +578,37 @@ fn main() {
             }));
         }
 
+        if matches.is_present("enable-mqtt-integration") || matches.is_present("enable-all") {
+            log::info!("Enabling MQTT integration");
+            let bind_addr_mqtt = server.mqtt_integration.clone().into();
+            let kafka = server.kafka.clone();
+            threads.push(std::thread::spawn(move || {
+                let config = drogue_cloud_mqtt_integration::Config {
+                    health: None,
+                    oauth,
+                    disable_tls: true,
+                    cert_bundle_file: None,
```

Comment:
Can this use the same logic as for the http endpoint to enable TLS?

Reply:
Ah sure. I copied the logic from the MQTT endpoint. I will fix both.